### PR TITLE
Add guidelines link under "Add a recipe" section on website.

### LINKED ISSUE #138: Link to guidelines from "Add a recipe" section
As suggested by @wholmgren in this [comment](https://github.com/conda-forge/staged-recipes/pull/616#issuecomment-219543636), it would be great if we had a link to the guidelines from the website. This should help a user see what is needed and recommended for recipes when they are learning how to add them.

### PR DESCRIPTION
**Pull request by ssurbhi560**
fixes #138 

PR Checklist:

- [ ] make all edits to the docs in the `src` directory, not in `docs` or in the html files
There are changes in html files.
- [x] note any issues closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)

### CODE REVIEW

File: index.html.tmpl [hunk 1]
```diff
@@ -246,10 +246,11 @@
...
-                          Add a new conda recipe in a new "recipes/[your-package-name]" directory.
-                          There is an example of a well-written recipe in "recipes/example" you can copy.
-                          <a href="docs/maintainer/adding_pkgs.html#the-recipe-meta-yaml">Further guidance on writing good recipes</a>.
-                          <a href="docs/maintainer/adding_pkgs.html#running-tests-locally-for-staged-recipes">You can also test the recipe locally.</a>
+                          Add a new conda recipe in a new "recipes/[your-package-name]" directory. Please make sure to follow these
+                          <a href="https://conda-forge.org/docs/orga/guidelines.html">guidelines</a>.
+                          You can also refer to an example of a well-written recipe in "recipes/example".
+                          Further guidance on writing good recipes can be found <a href="docs/maintainer/adding_pkgs.html#the-recipe-meta-yaml">here</a>.
+                          You can also <a href="docs/maintainer/adding_pkgs.html#running-tests-locally-for-staged-recipes">test a recipe locally.</a>
```
Comment (beckermr) [inline index.html.tmpl:253]:
```suggestion
                          You can also <a href="docs/maintainer/adding_pkgs.html#running-tests-locally-for-staged-recipes">test your recipe locally.</a>
```

Comment (ssurbhi560) [reply index.html.tmpl]: Done! Thanks for the suggestion! :)

**Comment by ssurbhi560** [comment]
@beckermr can you please review it again, I have made the changes suggested by you. Do let me know if there are any more changes to be made!

**Comment by beckermr** [comment]
Looks great!

File: index.html.tmpl [hunk 2]
```diff
@@ -246,10 +246,11 @@
...
-                          Add a new conda recipe in a new "recipes/[your-package-name]" directory.
-                          There is an example of a well-written recipe in "recipes/example" you can copy.
-                          <a href="docs/maintainer/adding_pkgs.html#the-recipe-meta-yaml">Further guidance on writing good recipes</a>.
-                          <a href="docs/maintainer/adding_pkgs.html#running-tests-locally-for-staged-recipes">You can also test the recipe locally.</a>
+                          Add a new conda recipe in a new "recipes/[your-package-name]" directory. Please make sure to follow these
+                          <a href="https://conda-forge.org/docs/orga/guidelines.html">guidelines</a>.
+                          You can also refer to an example of a well-written recipe in "recipes/example".
+                          Further guidance on writing good recipes can be found <a href="docs/maintainer/adding_pkgs.html#the-recipe-meta-yaml">here</a>.
+                          You can also <a href="docs/maintainer/adding_pkgs.html#running-tests-locally-for-staged-recipes">test your recipe locally.</a>
```
Comment (kathatherine) [inline index.html.tmpl:253]: The two instances of "you can also" are a little repetitive here. We could change the second one to "Test your recipe locally using [these instructions](LINK)."

**Review by kathatherine — APPROVED** [review]
Looks great.

**Comment by kathatherine** [comment]
@beckermr Looks like this one can be merged in, if that still works for you.

**Pull request merged**